emperor/support_files/js/controller.js

```diff
@@ -141,7 +192,11 @@ EmperorController.prototype.buildUI = function() {
     enableCellNavigation: true,
     forceFitColumns: true
   };
+  /**
```

Comment:
It doesn't seem to be addressed i.e. the line "Updates slick-grid ..." is still missing a `* ` at the beginning of the line.